### PR TITLE
Add Ferdium client 

### PR DESCRIPTION
**Pull request by lhw**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. 
- [x]  I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
  - appstream metadata ist not in yet. Undecided on necessity of version fields

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

Essentially just a copy of https://github.com/flathub/com.getferdi.Ferdi/ with some corrections for the new maintained fork. Old fork will get an EOL and rebase as soon as ferdium is available.

### CODE REVIEW

**Comment by lhw** [comment]
bot, build org.ferdium.Ferdium

**Comment by lhw** [comment]
bot, build org.ferdium.Ferdium

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: no build from source to support aarch64?

File: org.ferdium.Ferdium.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r usr/* opt/* /app
+      - chmod -R a-s,go+rX,go-w /app/Ferdium
+      - rm -r /app/share/icons/hicolor/1024x1024
+      - desktop-file-edit --set-key=Exec --set-value='ferdium %U' /app/share/applications/ferdium.desktop
+      - install ferdium.sh /app/bin/ferdium
+      - install -Dm644 org.ferdium.Ferdium.metainfo.xml /app/share/metainfo/org.ferdium.Ferdium.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/ferdium/ferdium-app/releases/download/v6.0.0-beta.1/ferdium_6.0.0-beta.1_amd64.deb
```
Comment (hfiguiere) [inline org.ferdium.Ferdium.yaml:61]: how about adding aarch64 support?

image

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.ferdium.Ferdium.yaml (hunk 2 above)
Comment (lhw) [reply org.ferdium.Ferdium.yaml:61]: Sounds like a good idea. Will check on that later today.

File: flathub.json (hunk 1 above)
Comment (lhw) [reply flathub.json:2]: Maybe. But from what I have seen in the electron build guide it's a major hassle. But won't put it off the table

Comment (hfiguiere) [reply flathub.json:2]: since there is an upstream aarch64 package it can work too. Just that usually there is none.

**Comment by lhw** [comment]
bot, build org.ferdium.Ferdium

File: flathub.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (hfiguiere) [inline flathub.json:1]: you can just remove this file. `aarch64` and `x86_64` are the only two arch supported anyway.

File: org.ferdium.Ferdium.yaml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        sha256: 89445c022abae41341ea9c6963e5a56b53b08391432c11bde125900998cc4f50
+        x-checker-data:
+          type: html
+          url: https://ferdium.org/download/
+          version-pattern: 'Current version: ([\d\.-]+(\-beta\.\d+))?'
+          url-template: https://github.org.ferdium/ferdium-app/releases/download/v$version/ferdium_${version}_amd64.deb
+      - *source
```
Comment (hfiguiere) [inline org.ferdium.Ferdium.yaml:93]: the trick here would be to have ONE module, and mark each source as `only-arches`. And avoid these "referemces" that make everything hard to follow. Bonus point: the manifest is way shorter.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.ferdium.Ferdium.yaml (hunk 4 above)
Comment (lhw) [reply org.ferdium.Ferdium.yaml:93]: Oh neat. I was experimenting with anchors and noticed that merge keys didnt work either so I stopped right there. Will update it real quick

**Comment by hfiguiere** [comment]
bot, build org.ferdium.Ferdium
